api/envoy/extensions/wasm/v3/wasm.proto

```diff
@@ -50,7 +49,7 @@ message VmConfig {
   // [#comment: TODO: add ref for details.]
   string vm_id = 1;
 
-  // The Wasm runtime type.
+  // The Wasm runtime type, default to wasm engine used at envoy build-time.
```

Comment:
Nit: `default` -> `defaults` and `envoy` -> `Envoy`.

Reply:
Fixed.